dojo/db_migrations/0068_risk_acceptance.py

```diff
@@ -0,0 +1,137 @@
+        ),
+        migrations.AddField(
+            model_name='risk_acceptance',
+            name='decision',
+            field=models.CharField(choices=[('A', 'Accept'), ('V', 'Avoid'), ('C', 'Compensate'), ('R', 'Reduce'), ('T', 'Transfer')], default='A', help_text='Risk treatment decision by risk owner', max_length=2),
+        ),
+        migrations.AlterField(
+            model_name='risk_acceptance',
+            name='path',
+            field=models.FileField(blank=True, null=True, upload_to='risk/%Y/%m/%d', verbose_name='Proof'),
+        ),
+        migrations.AddField(
+            model_name='risk_acceptance',
+            name='reactivate_expired',
+            field=models.BooleanField(default=True, help_text='Reactivate findings when risk acceptance expires?', verbose_name='Reactivate findings on expiration'),
+        ),
+        migrations.AddField(
+            model_name='product',
+            name='enable_full_risk_acceptance',
+            field=models.BooleanField(default=True, help_text='Allows full risk acceptanc using a risk acceptance form, expiration date, uploaded proof, etc.'),
+        ),
+        # convert existing products to have simple risk acceptance enabled as before
+        migrations.AddField(
+            model_name='product',
+            name='enable_simple_risk_acceptance',
+            field=models.BooleanField(default=True, help_text='Allows simple risk acceptance by checking/unchecking a checkbox.'),
+        ),
+        # new products will have simple risk acceptance disabled
+        migrations.AlterField(
+            model_name='product',
+            name='enable_simple_risk_acceptance',
+            field=models.BooleanField(default=False, help_text='Allows simple risk acceptance by checking/unchecking a checkbox.'),
+        ),
+        migrations.AddField(
+            model_name='jira_project',
+            name='risk_acceptance_expiration_notification',
+            field=models.BooleanField(default=False, verbose_name='Send Risk Acceptance expiration notifications as comment?'),
+        ),
+        migrations.AddField(
+            model_name='risk_acceptance',
+            name='expiration_date_handled',
+            field=models.DateTimeField(blank=True, default=None, help_text='(readonly) When the risk acceptance expiration was handled (manually or by the daily job.', null=True),
+        ),
+        migrations.AddField(
+            model_name='risk_acceptance',
+            name='expiration_date_warned',
+            field=models.DateTimeField(blank=True, default=None, help_text='(readonly) When the risk acceptance expiration was warned for by the daily job.', null=True),
```

Comment:
Rephrase to "Date at which notice about the risk acceptance expiration was sent"?